running pytest
Searching for pytest<5.0.0,>=4.6.4
Best match: pytest 4.6.11
Processing pytest-4.6.11-py3.6.egg

Using /code/modules/weko-schema-ui/.eggs/pytest-4.6.11-py3.6.egg
running egg_info
writing weko_schema_ui.egg-info/PKG-INFO
writing dependency_links to weko_schema_ui.egg-info/dependency_links.txt
writing entry points to weko_schema_ui.egg-info/entry_points.txt
writing requirements to weko_schema_ui.egg-info/requires.txt
writing top-level names to weko_schema_ui.egg-info/top_level.txt
reading manifest file 'weko_schema_ui.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
adding license file 'LICENSE'
adding license file 'AUTHORS.rst'
writing manifest file 'weko_schema_ui.egg-info/SOURCES.txt'
running build_ext
============================= test session starts ==============================
platform linux -- Python 3.6.15, pytest-4.6.11, py-1.11.0, pluggy-0.13.1
rootdir: /code/modules/weko-schema-ui, inifile: pytest.ini, testpaths: docs, tests, weko_schema_ui
plugins: celery-4.4.4, pep8-1.0.6, flask-0.15.1, cov-2.10.1, invenio-1.3.4
collected 39 items / 1 errors / 38 selected

==================================== ERRORS ====================================
___________ ERROR collecting weko_schema_ui/serializers/__init__.py ____________
weko_schema_ui/serializers/__init__.py:58: in <module>
    BibTexSerializer = WekoBibTexSerializer()
weko_schema_ui/serializers/WekoBibTexSerializer.py:130: in __init__
    self.__ns = cache_schema('jpcoar_mapping').get('namespaces')
E   AttributeError: 'NoneType' object has no attribute 'get'
=============================== warnings summary ===============================
.eggs/pytest-4.6.11-py3.6.egg/_pytest/mark/structures.py:335
.eggs/pytest-4.6.11-py3.6.egg/_pytest/mark/structures.py:335
.eggs/pytest-4.6.11-py3.6.egg/_pytest/mark/structures.py:335
  /code/modules/weko-schema-ui/.eggs/pytest-4.6.11-py3.6.egg/_pytest/mark/structures.py:335: PytestUnknownMarkWarning: Unknown pytest.mark.pep8 - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/latest/mark.html
    PytestUnknownMarkWarning,

/home/invenio/.virtualenvs/invenio/lib/python3.6/site-packages/invenio_indexer/cli.py:96
  /home/invenio/.virtualenvs/invenio/lib/python3.6/site-packages/invenio_indexer/cli.py:96: DeprecationWarning: 'resultcallback' has been renamed to 'result_callback'. The old name will be removed in Click 8.1.
    @queue.resultcallback()

/home/invenio/.virtualenvs/invenio/lib/python3.6/site-packages/flask_oauthlib/contrib/cache.py:3
  /home/invenio/.virtualenvs/invenio/lib/python3.6/site-packages/flask_oauthlib/contrib/cache.py:3: DeprecationWarning: 'werkzeug.contrib.cache' is deprecated as of version 0.15 and will be removed in version 1.0. It has moved to https://github.com/pallets/cachelib.
    from werkzeug.contrib.cache import NullCache, SimpleCache, FileSystemCache

-- Docs: https://docs.pytest.org/en/latest/warnings.html

---------- coverage: platform linux, python 3.6.15-final-0 -----------
Name                                                 Stmts   Miss  Cover   Missing
----------------------------------------------------------------------------------
weko_schema_ui/__init__.py                               4      0   100%
weko_schema_ui/admin.py                                 24      8    67%   39-40, 48, 56-68
weko_schema_ui/api.py                                   58     39    33%   50-60, 72-77, 90-95, 105-110, 119-123, 146-165, 170-174
weko_schema_ui/bundles.py                                3      0   100%
weko_schema_ui/config.py                                23      0   100%
weko_schema_ui/ext.py                                   31     19    39%   36-37, 44-46, 54-61, 72-73, 83-88, 95-97
weko_schema_ui/mappings/__init__.py                      1      0   100%
weko_schema_ui/mappings/v2/__init__.py                   1      0   100%
weko_schema_ui/mappings/v6/__init__.py                   1      0   100%
weko_schema_ui/models.py                                23      0   100%
weko_schema_ui/permissions.py                            4      0   100%
weko_schema_ui/rest.py                                 128     98    23%   51-54, 65-151, 161-171, 179, 188-278, 287-310, 320-325, 335-340
weko_schema_ui/schema.py                              1133   1083     4%   47-53, 58, 62-209, 228-251, 255-269, 279-302, 306-375, 387-396, 400-977, 986-1582, 1586-1597, 1605-1637, 1644-1656, 1660-1687, 1698-1704, 1708-1794, 1809-1819, 1827-1834, 1839-1841, 1844, 1855-1863, 1875-1887, 1898, 1903-1934
weko_schema_ui/serializers/WekoBibTexSerializer.py     340    257    24%   131-138, 175-191, 199-206, 215-224, 233-239, 248-258, 267-279, 288-300, 309-319, 328-334, 343-349, 358-364, 373-380, 389-395, 404-409, 418-424, 436-465, 477-481, 491-495, 504-513, 522-589, 597-607, 616-707, 718-726, 737-753, 763-766
weko_schema_ui/serializers/__init__.py                  18      0   100%
weko_schema_ui/serializers/wekoxml.py                   14      3    79%   60, 71-75
weko_schema_ui/utils.py                                 59     42    29%   48-50, 61-67, 80-90, 101-127, 132-135, 140-151, 156-158
weko_schema_ui/version.py                                2      0   100%
weko_schema_ui/views.py                                  3      0   100%
----------------------------------------------------------------------------------
TOTAL                                                 1870   1549    17%

!!!!!!!!!!!!!!!!!!! Interrupted: 1 errors during collection !!!!!!!!!!!!!!!!!!!!
===================== 5 warnings, 1 error in 5.20 seconds ======================
